src/backend/identity/index.ts

```diff
@@ -35,22 +35,23 @@ export interface User {
 }
 
 /**
- * Defines a notification to send to a user’s Koji account.
- * Send notifications with [[pushNotificationToOwner]], for the user who created the Koji, or [[pushNotificationToUser]], for a user who interacts with the Koji and has granted the appropriate authorization.
+ * Defines a notification to send to a user account.
```

Comment:
```suggestion
 * Defines a notification to send to a user's Koji account.
```
The notifications are sent to the user's account on the Koji platform, so reverting to this clarification.

Reply:
Reverted.